R/downloadFcsFiles.R

```diff
@@ -0,0 +1,94 @@
+#' Download multiple FCS files
+#'
+#' Downloads multiple files bundled into a ZIP archive.
+#'
+#' @param experimentId ID of experiment.
+#' @param format Specifies the file format ("fcs", "tsv (with header)" or
+#' "tsv (without header)")
+#' @param destination [String] Write the files to the specified destination.
+#' @param overwrite [Logical] Allows a destination file to be overwritten.
+#' @param fcsFileIds [Array<String>] Optional. Defaults to returning all files
```

Comment:
all non-control* files

[The current API docs](https://docs.cellengine.com/api/#download-multiple-files-or-populations-fcs-or-tsv) have this. Is this text from an old version or the OpenAPI file maybe? (Asking because we need to make sure all of the other param strings are correct.)